keps/693-multikueue/README.md

```diff
@@ -0,0 +1,220 @@
+# KEP-693: MultiKueue
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+Introduce an new AdmissionCheck (called MultiKueue) with dedicated API
+and controller that will provide multi-cluster capabilities to Kueue.
+
+## Motivation
+Many of Kueue's users are running multiple clusters and would like to
+have a way to easily distribute batch jobs across them to keep all of
+them utilized. Without a global distribution point, some clusters may
+get less jobs they are able to process, while the others get more, leading
+to underutilization and higher costs.
+
+### Goals
+* Allow Kueue distribute batch jobs across multiple clusters,
+while maintaining the specified quota limits.
+* Provide users with a single entry point through which the jobs
+can be submitted and monitored, just like they were running in
+a single cluster.
+* Be compatible with all Kueue's features (priorities, borrowing, preemptions, etc)
+and most of integrations.
+
+### Non-Goals
+* Solve storage problem. It is assumed that the distributed jobs are 
+either location-flexible or are copying the data as a part of the 
+startup process.
+* Synchronize configuration across the clusters. It is expected that the 
+user will create the appropriate objects, roles and permissions
+in the clusters (manually, using gitops or some 3rd-party tooling).
+* Set up authentication between clusters.
+
+## Proposal
+
+Introduce MultiKueue AdmissionCheck, controller and configuration API. 
+
+Establish the need for a designated management cluster.
```

Comment:
WDYT about the `leader` for the name, instead of the management cluster? Leader is used in the WIP PR https://github.com/kubernetes-sigs/kueue/pull/1360, and I like this name, but no strong opinion, it would be just good to align the vocab between the KEP and the implementation.

Reply:
Leader has some meaning in K8S. Leaders are elected kind-of at random from more-or-less equal candidates. Here it is not the case.